parsons/action_network/action_network.py

```diff
@@ -169,6 +169,32 @@ def get_person_attendance(self, person_id, attendance_id):
         """
         return self.api.get_request(f"people/{person_id}/attendances/{attendance_id}")
 
+    def create_attendance(self, event_id, payload):
+        """
+        `Args:`
+            event_id: the unique id of the event
+            payload: the payload for creating the event attendance
+        `Returns:`
+            A JSON response after creating the event attendance
+        """
+        return self.api.post_request(f"events/{event_id}/attendances", payload)
```

Comment:
I'm noticing now that a lot of these methods just say "the payload". I imagine that the payloads for a lot of these items are very detailed and we don't want to put all that in the docstrings, but perhaps we could link to the payload details? For example this one could link to https://actionnetwork.org/docs/v2/attendances

Reply:
Hi @shaunagm! 
I hope that is ok I've added a "Documentation Reference" for each function for example:
        """
        `Args:`
            tag_id:
                The unique id of the tag
            payload:
                The payload for creating the tagging
                {
                    "_links" : {
                        "osdi:person" : { "href" : "https://actionnetwork.org/api/v2/people/id" }
                    }
                }
        `Returns:`
            A JSON response after creating the tagging
        **`Documentation Reference`:
            https://actionnetwork.org/docs/v2/taggings**
        """
Please let me know if this is acceptable or if you prefer another way to attach a reference from the documentation!

Reply:
This is perfect, thanks!